p2p/simulations/network.go

```diff
@@ -210,6 +221,24 @@ func (self *Conn) event(up, rev bool) *ConnEvent {
 	}
 }
 
+type Msg struct {
+	One   *adapters.NodeId `json:"one"`
+	Other *adapters.NodeId `json:"other"`
+	Code  uint64
```

Comment:
I'm sorry I'm not sure what you mean; the equivalent structs export to lowercase too?